config_builder.go

```diff
@@ -911,10 +917,6 @@ type DatabaseInstances struct {
 	// for native SQL queries for tables that already support it. This may
 	// be nil if the use-native-sql flag was not set.
 	NativeSQLStore *sqldb.BaseDB
-
-	// AuxLeafStore is an optional data source that can be used by custom
-	// channels to fetch+store various data.
-	AuxLeafStore fn.Option[lnwallet.AuxLeafStore]
```

Comment:
why was this here / being removed now?

Reply:
This was just in the wrong place, moved it to the `AuxComponents` sub struct.